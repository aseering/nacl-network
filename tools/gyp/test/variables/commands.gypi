# This file is included from commands.gyp to test evaluation order of includes.
{
  'variables': {
    'included_variable': 'XYZ',
  },
  'targets': [
    {
      'target_name': 'dummy',
      'type': 'none',
    },
  ],
}
