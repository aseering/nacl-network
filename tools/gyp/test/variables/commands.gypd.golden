{'_DEPTH': '.',
 'included_files': ['commands.gyp', 'commands.gypi'],
 'targets': [{'actions': [{'action': ['echo',
                                      '"3.14159265359 ABCD"',
                                      'ABCD letters_list'],
                           'action_name': 'test_action',
                           'inputs': ['3.14159265359 ABCD'],
                           'outputs': ['ABCD', 'letters_list'],
                           'variables': {'var7': 'letters_list'}}],
              'configurations': {'Default': {}},
              'default_configuration': 'Default',
              'target_name': 'foo',
              'type': 'none',
              'variables': {'var1': '3.14159265359',
                            'var2': '3.14159265359 ABCD',
                            'var3': 'ABCD',
                            'var4': 'ABCD',
                            'var5': 'letters_',
                            'var6': 'list'}},
             {'configurations': {'Default': {}},
              'default_configuration': 'Default',
              'target_name': 'dummy',
              'type': 'none'}],
 'variables': {'check_included': 'XYZ',
               'check_lists': ['XYZ', 'ABCDEFGHIJK'],
               'included_variable': 'XYZ',
               'letters_list': 'ABCD',
               'other_letters': 'ABCDEFG',
               'pi': 'import math; print math.pi',
               'third_letters': 'ABCDEFGHIJK'}}
